qemu/tests/driver_in_use.py

```diff
@@ -33,6 +33,8 @@ def check_bg_running(target_process):
         list_cmd = params.get("list_cmd", "wmic process get name")
         output = session.cmd_output_safe(list_cmd, timeout=60)
         process = re.findall(target_process, output, re.M | re.I)
+        if params['os_type'] == 'linux':
+            process = session.cmd_output('pgrep -x %s' % target_process)
```

Comment:
The command does not work for linux guest during migration.

Reply:
Thanks, have updated.